gamemodes/terrortown/gamemode/client/cl_help/populate_appearance.lua

```diff
@@ -0,0 +1,554 @@
+local materialIcon = Material("vgui/ttt/vskin/helpscreen/appearance")
+
+local function PopulateGeneralPanel(parent)
+	local form = CreateForm(parent, "header_global_color")
+
+	form:MakeHelp({
+		label = "help_color_desc"
+	})
+
+	-- store the reference to the checkbox in a variable
+	-- because the other settings are enabled based on
+	-- the state of this checkbox
+	local enbColor = form:MakeCheckBox({
+		label = "label_global_color_enable",
+		initial = GLAPP.ShouldUseGlobalFocusColor(),
+		onChange = function(_, value)
+			GLAPP.SetUseGlobalFocusColor(value)
```

Comment:
Because convars will be removed and I designed this system in a way that it doesn't matter how the data is stored

Reply:
But since we are using ConVars for other settings like Crosshair etc. it would be more consistent to use ConVars here too and refactor it later alltogether.

Reply:
But since it is changed in one of the next PRs, I don't see a reason to use convars here. Besides it is more readable to call one function that to set four convars